bundles/org.openhab.binding.ipcamera/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -17,6 +17,9 @@
 			<add-channel id="enablePush">
 				<type>ipcamera:enablePush</type>
 			</add-channel>
+			<add-channel id="reboot">
+				<type>ipcamera:reboot</type>
+			</add-channel>
```

Comment:
Add this new channel to a v3 version:
`<instruction-set targetVersion="3">`

Also update the thingTypeVersion to 3